Tests/Resources/TestBundle/Security/ResourceVoter.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Symfony\Cmf\Bundle\ResourceRestBundle\Tests\Resources\TestBundle\Security;
+
+use Symfony\Component\Security\Core\Authentication\Token\TokenInterface;
+use Symfony\Component\Security\Core\Authorization\Voter\Voter;
+
+class ResourceVoter extends Voter
+{
+    protected function supports($attribute, $subject)
+    {
+        return in_array($attribute, ['CMF_RESOURCE_READ', 'CMF_RESOURCE_WRITE']);
+    }
+
+    protected function voteOnAttribute($attribute, $subject, TokenInterface $token)
+    {
+        if ('/cms/public' !== substr($subject['path'], 0, 11)) {
```

Comment:
The requested resource path to the controller. I have no idea what you mean with "configured". 

*Please be aware that you're now looking at test fixture code.*